presto-docs/src/main/sphinx/themes/presto/static/presto.css_t

```diff
@@ -26,12 +26,12 @@ body {
     color: #222222;
 }
 
-a:link, a:visited {
+a:link, a:visited, a.headerlink {
   font-weight: bold;
   text-decoration: none;
   color: #44af5c; }
 
-a:hover, a:active {
+a:hover, a:active, a.headerlink:hover {
```

Comment:
redundant?

Reply:
No, there is a specific style for `a.headerlink:hover` in the included `haiku.css` (comes from the theme so no easy way to change it ... need to clean all that up sometime)